gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/amqp/ServiceFailureIT.java

```diff
@@ -137,10 +142,10 @@ public void When_ReceivingEhrExtract_WithMhsOutboundServerError_Expect_Migration
 
     @Test
     public void When_ReceivingCOPC_WithMhsOutboundServerError_Expect_MessageSentToDLQ() {
-        doThrow(MhsServerErrorException.class)
+        doThrow(DataFormatException.class)
```

Comment:
What's this?

Reply:
Runtime exception type.

Reply:
As a developer, I'd have no idea what this test is doing then.

Reply:
The description talks about `WithMhsOutboundServerError`

Reply:
Oh, yeah, I will adjust the description, completely forgot about it. Thanks